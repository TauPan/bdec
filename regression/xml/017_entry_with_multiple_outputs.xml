<protocol>
    <!--
      There was a problem with an entry that had multiple outputs from a
      single child (in this test, 'c' has outputs for 'a' and 'b', both of
      which are passed through 'problem').
      -->
    <sequence name="spec">
        <sequence name="problem">
            <sequence name="c">
                <field name="a" length="8" type="integer" />
                <field name="b" length="8" type="integer" />
            </sequence>
        </sequence>
        <field name="d" length="${problem.c.a}" />
        <field name="e" length="${problem.c.b}" />
    </sequence>

</protocol>
