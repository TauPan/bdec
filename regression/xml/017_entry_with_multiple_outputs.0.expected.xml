<spec><problem><c><a>6</a><b>2</b></c></problem><d>111100</d><e>01</e></spec>